### ISSUE TITLE
Type error in create-map/index.d.ts file

### ISSUE THREAD

**Issue by davidmz** [nanostores/nanostores#54]
The `create-map/index.d.ts` causes the following error in my repo:
```
node_modules/nanostores/create-map/index.d.ts:2:38 - error TS2322: Type 'K' is not assignable to type 'string | number | symbol'.
  Type 'K' is not assignable to type 'symbol'.

2 type Get<T, K> = Extract<T, { [K1 in K]: any }>[K];
```

The same error I see in TypeScript playground ([here](https://www.typescriptlang.org/play?#code/C4TwDgpgBA4hwB4AqAaKBpAfFAvFAogB7ABOAhgMaKpQDeUA2ugIxQCWAdhgLoBcUZDiCgBfTE24BuAFBA)).

**Comment by davidmz** [comment]
Probably this string should look like `type Get<T, K *extends string*> = Extract<T, { [K1 in K]: any }>[K];` ?

**Comment by ai** [comment]
It seems like this issue is related to the latest @AleksandrSl and @Beraliv fixes

**Comment by Beraliv** [comment]
> It seems like this issue is related to the latest @AleksandrSl and @Beraliv fixes

Let me quickly check

**Comment by Beraliv** [comment]
I can fix it

The example: https://tsplay.dev/N7gdqN

**Comment by ai** [comment]
Great! Send PR.

**Comment by ai** [comment]
The fix was released in 0.4.7
